tests/spec/w3c/linter-rules/required-sections-spec.js

```diff
@@ -61,8 +61,8 @@ describe("w3c — required-sections", () => {
     expect(warnings).toHaveSize(2);
   });
 
-  it("doesn't lint non-rec-track docs", async () => {
-    for (const specStatus of [...noTrackStatus, "ED"]) {
+  for (const specStatus of [...noTrackStatus, "ED"]) {
```

Comment:
Can we include "ED" in noTrackStatus?

Reply:
I'll look at doing that once I get around a rewriting the mess that is headers.js ... there were "special cases".